data.json

```diff
@@ -98,6 +98,15 @@
       "linkedin": "https://www.linkedin.com/in/ayushkhandelwal28/",
       "quote": "Always Learning"
     }
```

Comment:
```suggestion
    },
```

Reply:
I made the changes . Are the visible to you ?